pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -627,7 +628,9 @@ template <class StateVectorT> void lightningClassBindings(py::module_ &m) {
     /* Observables submodule */
     py::module_ obs_submodule =
         m.def_submodule("observables", "Submodule for observables classes.");
-    registerObservables<StateVectorT>(obs_submodule);
+    // registerBackendAgnosticObservables<StateVectorT>(obs_submodule);
```

Comment:
Are there any plans for that? Or could we remove it?

Reply:
Nope, thanks.